pkg/cmd/repo/list/list.go

```diff
@@ -86,20 +78,19 @@ func NewCmdList(f *cmdutil.Factory, runF func(*ListOptions) error) *cobra.Comman
 	}
 
 	cmd.Flags().IntVarP(&opts.Limit, "limit", "L", 30, "Maximum number of repositories to list")
-	cmd.Flags().BoolVar(&flagPrivate, "private", false, "Show only private repositories")
-	cmd.Flags().BoolVar(&flagPublic, "public", false, "Show only public repositories")
```

Comment:
Removing these flags breaks backwards compatibility. I would suggest leaving them in and having them be the equivalent of passing `--visibility private/public`.

Reply:
I agree we should not remove these, which is why I think it would actually be less confusing to our users to just add a `--internal` flag instead of a more generic `--visibility` flag. Not a super strong opinion on this one but thought I would chime in.